assets/css/style.css

```diff
@@ -375,25 +434,52 @@
 
   .box {
     display: grid;
+    grid-template-columns: 1fr;
     grid-template-rows: 20fr 30fr;
-    grid-template-columns: 100%;
-    height: 316px;
     border: 1px solid #c1c7d0;
     background: #ebf0ee;
     border-radius: 8px;
+    width: 369px;
+    height: 316px;
   }
 
-  .form {
+  .contact {
+    margin-top: 1px;
+    background-image: url(../img/desk_contact.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    margin-left: 0;
+    margin-right: 0;
     display: flex;
     flex-direction: column;
     justify-content: center;
-    gap: 20px;
-    background-image: url(../img/imgfoot.svg);
+  }
+
+  .contact-container {
```

Comment:
- [ ] Good job so far 👍👍 I noticed that in about section, the input fields are overflowing and the section content is hidden ⚠ Please consider fixing this according to the design guidelines to get a design close to the FIgma template as mentioned in the project requirements

![image](https://github.com/Graciano1997/portofolio-graciano/assets/106119105/5d1d7138-1ccc-4446-a467-c11749d8a2ff)